apps/web/src/server/applications/create/applicant/applications-create-applicant.controller.js

```diff
@@ -355,9 +359,9 @@ export async function updateApplicationsCreateApplicantWebsite(
  * @type {import('@pins/express').RenderHandler<ApplicationsCreateApplicantTelephoneNumberProps, {}, {}, {}, {}>}
  */
 export async function viewApplicationsCreateApplicantTelephoneNumber(req, response) {
-	const { applicationId } = response.locals;
+	const { applicationId, applicantId } = response.locals;
 
-	const applicant = await getNthApplicant(applicationId, 0);
+	const applicant = await getApplicantById(applicationId, applicantId, '{"applicants": true}');
```

Comment:
see below about the query always being `applicant` for this service

Reply:
it is different for the address call

Reply:
oh, right.